andela/AndelaEatsUI
# #163082544 Default avatar for meals

#### What does this PR do?
- Add default avatar for meals
#### Description of Task to be completed?
- Choose and add to the image folder an appropriate image to be used as a default avatar.

#### How should this be manually tested?
- Add a meal item but do not upload an image
- A default image should be shown in place of the image
#### What are the relevant pivotal tracker stories?
- [#163082544](https://www.pivotaltracker.com/story/show/163082544)

#### Background Context
N/A
#### Screenshots (If Applicable)
![record](https://user-images.githubusercontent.com/39955261/51201442-20f97780-190d-11e9-892d-05a618b17ce8.gif)
